Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,118 @@
+
+h1 {
+	text-align: center;
+	line-height: 0.5;
+	font-size: 20px;
+	font-family: arial;
+	padding-bottom: 8px;
+}
+
+h2 {
+	text-align: center;
+	font-size: 35px;
+	line-height: 0.05;
+	font-family: arial;
+	padding-bottom: 30px;
+}
+
+h3 {
+	text-align: left;
+	margin-left: 10px;
+	font-size: 20px;
+	line-height: 0.05;
+	font-family: arial;
+	padding-bottom: 15px;
+
+}
+
+h4 {
+	text-align: left;
+	margin-left: 10px;
+	font-size: 15px;
+	font-weight: 500;
+	line-height: 0.05;
+	font-family: verdana;
+	padding-bottom: 15px;
+
+}
+
+/*i'm not entirely sure how to separate out pieces
+in the same line... for instance larger font, italics 
+and bold... but only "web developer"*/
+
+h5 {
+	text-align: left;
+	font-size: 13px;
+	font-family: times;
+	
+}
+
+/*alas, i cannot for the life of me figure out
+how to padd the second line in the content borders 
+to not run into the avatar image*/
+
+.body {
+	border-top: 1px solid #000;
+	margin-left: 70px;
+	margin-right: 70px;
+	padding-top: 50px;
+	padding-bottom: 20px;
+	border-bottom: 1px solid #000;
+}
+
+.avatar {
+	width: 250px; height:250px;
```

Comment:
Watch your formatting here — make sure you're consistent with your white-space and keeping rules on individual lines!